.github/workflows/rust.yml

```diff
@@ -24,29 +24,28 @@ jobs:
       - name: Running fuzzer
         env:
           DO_FUZZ: true
-          DO_LINT: true
         run: ./contrib/test.sh
 
-  bench_nightly:
-    name: Bench + Tests
+  Nightly:
+    name: Bench + Lint
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        rust:
-          - nightly
     steps:
       - name: Checkout Crate
         uses: actions/checkout@v2
       - name: Checkout Toolchain
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: ${{ matrix.rust }}
+          toolchain: nightly
           override: true
-      - name: Running cargo test
+      - name: Running bench
         env:
           DO_BENCH: true
         run: ./contrib/test.sh
+      - name: Running clippy
```

Comment:
nit: It says running Clippy over here. I think we mean rustfmt?

Reply:
Ha! I started all this because the `contrib/test.sh` used `DO_LINT` to run rustfmt, which, as you point out is wrong - then proceeded a few days later to do the same mistake - that's funny.